### DISCUSSION TITLE
How to Change Wifi Network?

### DISCUSSION THREAD

**Discussion by PrestonMcAfee** [gtxaspec/wz_mini_hacks#809] [Q&A]
This is probably obvious but if I want to put the camera on a different wifi network, how do I do it?  When I log in I don't see an entry for the network.  I'm hoping to avoid reinstalling the Wyze app.

**Comment by bjs-pdx** [comment]
Sorry to say but WiFi config is only via the Wyze App.  The mods only work on the cams after they've been configured in the Wyze App and connected to the WiFi network.

**Comment by PrestonMcAfee** [comment]
Thanks, good to know, I guess I'll download the Wyze app again.
